build.gradle

```diff
@@ -186,24 +220,26 @@ uploadArchives {
         }
     }
 
-    //Dependencies for pom generation
+    // Dependencies for pom generation
```

Comment:
Has it been retained on purpose?

Reply:
No commented out -- see the line below. Will check once the pom is published though it works just fine with local jar (and no other dependencies are actually required because this is a shadow jar).

Reply:
Yes, I meant why just comment out but not remove altogether. Ok.